src/components/views/messages/DateSeparator.tsx

```diff
@@ -141,11 +142,12 @@ export default class DateSeparator extends React.Component<IProps, IState> {
                 `found ${eventId} (${originServerTs}) for timestamp=${unixTimestamp} (looking forward)`,
             );
 
-            dis.dispatch({
+            dis.dispatch<ViewRoomPayload>({
                 action: Action.ViewRoom,
                 event_id: eventId,
                 highlighted: true,
                 room_id: roomId,
+                _trigger: null, // room doesn't change
```

Comment:
is there a difference between null and undefined for _trigger? If so, could comment here

Reply:
Should be undefined, the underlying type won't allow null but we don't have strict null checking. Will fix, thanks